<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>487</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1207</width>
    <height>487</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1213</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recursos Humanos</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>370</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;
 background-color: rgb(0, 170, 255);
  border-radius: 13px;

}

QPushButton:hover{
background-color: blue;
}

QPushButton:pressed{
  background-color: rgb(0, 255, 127);

}</string>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;
 background-color: rgb(0, 170, 255);
  border-radius: 13px;

}

QPushButton:hover{
background-color: blue;
}

QPushButton:pressed{
  background-color: rgb(0, 255, 127);

}</string>
       </property>
       <property name="text">
        <string>Alterar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  padding: 5px 15px;
  font-size: 15px;
  text-align: center;
  color: #fff;
 background-color: rgb(0, 170, 255);
  border-radius: 13px;

}

QPushButton:hover{
background-color: red;
}

QPushButton:pressed{
  background-color: rgb(0, 255, 127);

}</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>90</y>
      <width>185</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar Funcionário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
background-color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>272</y>
      <width>305</width>
      <height>35</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string>Inseri o nome</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
background-color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Cargo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>312</y>
      <width>305</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
background-color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Departamento</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>305</width>
      <height>35</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string>Inseri o nome</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
background-color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Introduza o Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>230</y>
      <width>305</width>
      <height>35</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string>Inseri o nome</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font-size: 13px;
text-align: center;
border-radius: 10px;
background-color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Área de Trabalho</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>193</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Área de Trabalho</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cargo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Departamento</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Ficheiro</string>
    </property>
    <addaction name="actionRecrutamento_Interno"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionRecrutamento_Interno">
   <property name="text">
    <string>Recrutamento Interno</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
